src/components/Theme/__tests__/ScreenLayout.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { h } from 'preact'
+import { mount } from 'enzyme'
+import ScreenLayout from '../ScreenLayout'
+
+describe('ScreenLayout', () => {
+  it('mounts correctly', () => {
+    const screen = mount(<ScreenLayout>Content</ScreenLayout>)
+    expect(screen.exists()).toBeTruthy()
```

Comment:
This should assert the correct children rendering

```suggestion
    expect(screen.exists()).toBeTruthy()
    expect(screen.text()).toEqual('Content')
```